data/grc/Erbse_1961_Theokrit-Eidyllia/stemma.gv

```diff
@@ -0,0 +1,17 @@
+digraph {
+archetyp [color="grey", label="α"] 
+hyparchetypbeta [color="grey", label="β"] 
+archetyp -> hyparchetypbeta 
+PAnt [label="P. Ant."] 
+archetyp -> PAnt 
+GroupLaur [label="Laur.", color="grey"] 
+GroupVat [label="Vat.", color="grey"] 
+1 [color="grey", label=""] 
+2 [color="grey", label=""] 
+hyparchetypbeta -> 1 
+1 -> K 
+1 -> GroupVat 
+hyparchetypbeta -> 2 
+2 -> GroupLaur 
```

Comment:
Ok, here too we have archetypes included without showing their descendants ! I am starting to understand that this is common practice among classicists.

Reply:
I was surprised as well. It's interesting to see the different traditions in each field.